README.md

```diff
@@ -2,11 +2,36 @@
 
 NOTE: The project is under development.
 
+## PREREQUISITES
 
-## Deploy Onload Operator
+1) Go version 1.21+
+2) The binaries of oc, kubectl present in $PATH
+3) Docker or a symbolic link to a API equivalent (e.g. podman) present in $PATH.
+4) The following exports.
+```text
+export KUBECONFIG=<Path to Kubeconfig for your deployment>
+export REGISTRY_LOCATION=<URL for your docker registry>
+export REGISTRY_PORT=$REGISTRY_PORT
+```
+5) A docker registry location that is accessible from the cluster.
+
+Do note the following deployment instructions assume seemless push/pull capabilities
```

Comment:
If the new content does line-wrapping, the patch should also change the existing content that doesn't do it. Otherwise, it looks inconsistent!